User-Stories/PO-PM-User-Stories/PO-user-stories.md

```diff
@@ -0,0 +1,60 @@
+#Product Owner (PO) / Project Manager (PM) user stories
+We assume this role as a principal role compared to a developer. Thus this player has the right to task the developer (agent) and influences the priotization of the agents work. Typically such a role is a team lead or project manager or in modern words a product owner. For ease of read, we will title the role PO, konwing that this simplification will not cover all details associated with the different role models. But from a compliance perspective the models can be treated as equivalent.
+
+##Responsibilites, tasks and role
+The ```primary goal``` of this role is to deliver a software solution while minimizing all related risks. This may comprise security, export controls or the incompliance with obligations resulting from the application of open source components. 
+
+Thus this role will be ```accountable``` for:
+* Execution of all reuired tasks to provide a risk minimized software solutions
+* Arrange and provide all relevnat tasks to cope with resulting obligations
+* Completeness of files/data/code to be assessed and delivered within the scope of this project
+
+This typically results in the follwing ```tasks```:
+* Ensure code is scanned on a regular basis - optimally as part of the CI/CD-chain
+* Ensure that _all_ components _and_ resources comprised within the solution will be captured/covered by the scans (incl. runtime components)
+* Initiate appropriate measures to cope with obligations and provide associated documentation
+* Initiate measures to cope with violations and warnings
+* Initiate approval requests
```

Comment:
should be automatically - the wording implies a manual step in my opinion

Reply:
I believe it will remain manual. there are many runs where no approval is required. And then there is the point in time, when the responsible wants to get an approval for his work. This is when he will trigger the approval request. How much of this approval then will be done manually is a different tasks.

Reply:
Yep, agree. But even or especially then, someone will make the decision to add the compliance stuff. Even if this will then create the compliance materials automatically we have one with the reponsibility to trigger the approval. And even if the approval will be automated, there will be someone who has to take the responsibility that the machine automating the task, will do it s job correctly. Thus we have an approval responsible.

Reply:
Agree, there needs to be someone who "signs the approval"